scripts/zones/Arrapago_Reef/mobs/Medusa.lua

```diff
@@ -1,28 +1,45 @@
 -----------------------------------
 -- Area: Arrapago Reef
 -- NPC:  Medusa
+-- TODO: resists, attack/def boosts
 -----------------------------------
 
 require("scripts/globals/titles");
+require("scripts/zones/Arrapago_Reef/MobIDs");
+require("scripts/zones/Arrapago_Reef/TextIDs");
 
 -----------------------------------
 -- onMobSpawn Action
 -----------------------------------
 
 function onMobSpawn(mob)
+    mob:setLocalVar("eeshpp", math.random(5,99)); -- Uses EES randomly during the fight
 end;
 
 -----------------------------------
 -- onMobEngaged Action
 -----------------------------------
 
-function onMobEngaged(mob,target)
+function onMobEngaged(mob, target)
+    target:showText(mob, MEDUSA_ENGAGE);
+    SpawnMob(ExonOne,   180):updateEnmity(target);
+    SpawnMob(ExonTwo,   180):updateEnmity(target);
+    SpawnMob(ExonThree, 180):updateEnmity(target);
+    SpawnMob(ExonFour,  180):updateEnmity(target);
+end;
 
-	SpawnMob(16998863,180):updateEnmity(target);
-	SpawnMob(16998864,180):updateEnmity(target);
-	SpawnMob(16998865,180):updateEnmity(target);
-	SpawnMob(16998866,180):updateEnmity(target);
+-----------------------------------
+-- onMobFight Action
+-----------------------------------
 
+function onMobFight(mob, target)
+    local HPP = mob:getHPP();
+    if (mob:getLocalVar("usedees") == 0) then
+        if (HPP <= mob:getLocalVar("eeshpp")) then
+            mob:useMobAbility(479); -- Eagle Eye Shot

```

Comment:
The value does change, for Maat they are totally different values, it could be different for other mobs too. The AI also needs to remember if it already used it's two hour, so it won't use it again. This `useMobAbility` doesn't do that.

All of this is already handled in the core, and as a long term solution (what if we discover, some animations were wrong? Now we have to fix multiple scripts instead of fixing one method) I feel adding a new method to initiate a mob two hour is just better. It's a no-brainer method call, you want the mob to two hour now? Ok call this method, done.